Granular routing in Spin 2.5 (#1265)
fermyon/developer: content/spin/v2/manifest-reference.md

| Name                    | Required?  | Type        | Value    | Example   |
|-------------------------|------------|-------------|----------|-----------|

> | `route`                 | Required   | String or Table     | The route which this component handles. Requests to the route will cause the component to execute. | |

Reviewer: ```suggestion
| `route`                 | Required   | String or table     | The route which this component handles. Requests to the route will cause the component to execute. | |
```

(just for consistency with existing occurrence)

> |                         |            | String     | This may be an exact route (`/example`), which matches on the given path, or a wildcard route indicated by the suffix `/...` (`/example/...`), which matches any route under this prefix. If two routes overlap, requests are directed to the matching route with the longest prefix - see [the HTTP trigger documentation](http-trigger) for details and examples. | `"/api/cart/..."` |

Reviewer: Do you want to mention single-segment wildcards here, or make the wildcard stuff more generic e.g.

```suggestion
|                         |            | String     | This may be an exact route (`/example`), which matches only the given path, or may include wildcards (`/example/:id` or `/example/...`). If two routes overlap, requests are directed to the matching route with the longest prefix. See [the HTTP trigger documentation](http-trigger) for details and examples. | `"/api/cart/..."` |
```
